src/indexing.jl

```diff
@@ -26,6 +26,9 @@ Base.start(::Value) = false
 Base.next(x::Value, state) = (x, true)
 Base.done(x::Value, state) = state
 
+# Values have the indexing trait of their wrapped type
+_axistrait_el(::Type{<:Value{T}}) where {T} = _axistrait_el(T)
```

Comment:
Could this be tested?

Reply:
It was being implicitly tested with the indexing tests but I added explicit tests. Thanks!